libs/components/theme/src/lib/styles/themes/modern/_buttons.scss

```diff
@@ -455,8 +456,11 @@
 
   .sky-btn-sm {
     padding: $sky-padding-half $sky-padding;
```

Comment:
I didn't tokenize it because datepicker overwrites the padding and as far as I know this is _only_ used in datepicker, so I'm not sure what other values we should give it. Do you remember why we decided to keep it?